lib/reporters/console.js

```diff
@@ -143,8 +144,8 @@ internals.Reporter.prototype.end = function (notebook) {
             if (test.err.actual !== undefined &&
                 test.err.expected !== undefined) {
 
-                var actual = JSON.stringify(test.err.actual, null, 2);
-                var expected = JSON.stringify(test.err.expected, null, 2);
+                var actual = StringifySafe(test.err.actual, null, 2);
```

Comment:
Thoughts on falling back to StringifySafe if the normal stringify fails?  (https://github.com/isaacs/json-stringify-safe/issues/4)

I know we aren't super concerned with perf in testing, but it does seem reasonable to only use StringifySafe when needed.